### PR TITLE
add information about kubernetes integration issue in 6.8.0 to the changelog

### PR DESCRIPTION
**Pull request by arbll**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,11 @@ Release Notes
 Prelude
 -------
 
+Please note that a critical bug identified in this release affecting the kubernetes integration.
+The severity of the issue has led us to remove the packages for the
+affected platform (**Docker**). If you have upgraded to this version,
+on **Docker** we recommend you downgrade to ``6.7.0``.
```
Comment (olivielpeau) [inline CHANGELOG.rst]: we could add that the `latest` tag points to 6.7.0 at the moment

**Review by remeh — DISMISSED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -8,6 +8,11 @@ Release Notes
 Prelude
 -------
 
+Please note that a critical bug has been identified in this release affecting the kubernetes integration.
```
Comment (olivielpeau) [inline CHANGELOG.rst]: this is a bit too vague and worrying IMO :)

let's be a bit more precise here, something like "the `KUBERNETES` environment variable would not enable the collection of kubelet metrics" maybe?

**Review by olivielpeau — COMMENTED** [review]
Left a comment, also worth a review from @DataDog/container-integrations

File: CHANGELOG.rst [hunk 3]
```diff
@@ -8,6 +8,11 @@ Release Notes
 Prelude
 -------
 
+Please note that a critical bug has been identified in this release that would prevent the kubernetes integration from collecting kublet metrics.
```
Comment (hkaj) [inline CHANGELOG.rst]: typo: `kublet` --> `kubelet`

File: CHANGELOG.rst [hunk 4]
```diff
@@ -8,6 +8,11 @@ Release Notes
...
+The severity of the issue has led us to remove the packages for the
+affected platform (**Docker**) and to make the `latest` tag point to the `6.7.0` release. If you have upgraded to this version,
+on **Docker** we recommend you downgrade to ``6.7.0``.
```
Comment (hkaj) [inline CHANGELOG.rst]: `on **Docker** or **Kubernetes**` I know Docker is the platform we target, but this applies even if people use k8s with a different runtime than docker.

Comment (arbll) [reply CHANGELOG.rst]: There are people that use the linux packages (and probably install with a package manager) but still enable the k8s integration right ? Would `If you have upgraded to this version of the containerized agent we recommend you downgrade to ``6.7.0``.` be less confusing ?

Comment (hkaj) [reply CHANGELOG.rst]: yeah that works

**Review by hkaj — DISMISSED** [review]

**Pull request merged**